_data/accessibility-tests/checkbox.yml

```
@@ -0,0 +1,54 @@
+title: Checkbox
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain checkbox requirements. 
+    summary_additional: The checkbox has a visible label above it to indicate its purpose. For example, "Select your state."
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons in a checkbox, all labels have short descriptions that are easy to understand and act on. 
```

@finekatie 
cc @alex-hull @amycole501 

Looks like radio button references were accidentally included here. Recommend we borrow language from the [Select component page](https://designsystem.digital.gov/components/select/accessibility-tests/): 
```suggestion
  - summary: Checkbox labels are descriptive, brief and clear.
    summary_additional: When you view or interact with the checkbox component on your page, all labels have short descriptions that are easy to understand and act on. 
```